mm2src/rpc_task/src/manager.rs

```diff
@@ -196,7 +196,14 @@ impl<Task: RpcTask> RpcTaskManager<Task> {
                         serde_json::to_value(new_status).expect("Serialization shouldn't fail."),
                     );
                     if let Err(e) = self.streaming_manager.broadcast_to(event, client_id) {
-                        warn!("Failed to send task status update to the client (ID={client_id}): {e:?}");
+                        match e {
+                            StreamingManagerError::UnknownClient => {
+                                // UnknownClient log seems pretty much useless so moving to trace
+                                // level
+                                trace!("Failed to send task status update to the client (ID={client_id}): {e:?}")
```

Comment:
we currently are applying default client id = 0 for every task managed request. and if the GUI never hooks to SSE, this error shows up (task asking to send updates to client 0 but client 0 isn't even connected/known).

such a log is generally useful, but it's due to the defaults we set right now, it will show up in unwanted times.

nit: we can put a todo asking to revert this back to warn level once we ditch using a default client (when migrating to WS).